lib/open-sesame.coffee

```diff
@@ -5,8 +5,12 @@ LEADING_SLASH_REGEX = /^\//
 OPEN_COMMAND        = 'open-sesame:open-file-under-cursor'
 
 getPathUnderCursor = ->
-  atom.workspace.getActiveTextEditor()?.getWordUnderCursor
-    wordRegex: PATH_REGEX
+  editor = atom.workspace.getActivePaneItem()
+  text = editor?.getSelectedText()
+  unless text.length > 0
```

Comment:
I used an unless construct as I'd seen it done in other Atom plugins, but I'm not sure if an if construct would be better here.